src/DI/WebpackExtension.php

```diff
@@ -47,6 +52,7 @@ public function __construct(bool $debugMode)
 		$this->defaults['debugger'] = interface_exists(Tracy\IBarPanel::class);
```

Comment:
- `debugger` is consistent with Nette's core extensions
- `macros` could be `latte` I guess, but it's something you won't typically need to change, so it's not really exposed to most of the people